vendor/vendor.json

```diff
@@ -1025,6 +1025,14 @@
 			"revision": "5bf94b69c6b68ee1b541973bb8e1144db23a194b",
 			"revisionTime": "2017-03-21T23:07:31Z"
 		},
+		{
```

Comment:
I wonder why this is added as I does not seem we use the "top level" gopsutil package anywhere. How did you "update" gopsutil?